### PR TITLE
translated configure-pod-initialization-2017-8-15.md

### PR DESCRIPTION
**Pull request by nsu700**
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to  
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 1]
```diff
@@ -15,7 +16,26 @@ application Container runs.
 {% endcapture %}
 
 {% capture steps %}
+-->
 
+---
+标题: 配置Pod初始化
```
Comment (rootsongjc) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: title 不要翻译，这个是文档编译时候用的。

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 2]
```diff
@@ -15,7 +16,26 @@ application Container runs.
...
+---
+
+{% capture overview %}
+这篇教程指导如何在应用运行之前使用初始容器来初始化一个Pod。
```
Comment (rootsongjc) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: 不用写“一个Pod”，直接写Pod就行了。Init container我觉得翻译成Init容器即可。

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 3]
```diff
@@ -28,7 +48,20 @@ Here is the configuration file for the Pod:
...
+## 创建一个含有初始容器的Pod
 
+在这个例子里，我们会创建一个包含一个应用容器和一个初始容器的Pod，这个初始
+容器会在应用容器开始前完成它的工作。
+
+这是Pod的配置文件:
+
+{% include code.html language="yaml" file="init-containers.yaml" ghlink="/docs/tasks/configure-pod-container/init-containers.yaml" %}
+
+在这个配置文件里，您可以看到Pod有一个初始容器和应用容器共享的存储卷。
```
Comment (rootsongjc) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: 英文单词前后要加空格。

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 4]
```diff
@@ -89,3 +170,18 @@ The output shows that nginx is serving the web page that was written by the init
...
 
 {% include templates/task.md %}
+-->
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+* 了解更多
+[Pod里容器的通讯](/docs/tasks/configure-pod-container/communicate-containers-same-pod/).
```
Comment (rootsongjc) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md]: Pod中的容器间通信

File: docs/tasks/configure-pod-container/configure-pod-initialization.md [hunk 5]
```diff
@@ -89,3 +170,18 @@ The output shows that nginx is serving the web page that was written by the init
...
 
 {% include templates/task.md %}
+-->
+
+{% endcapture %}
+
+{% capture whatsnext %}
+
+* 了解更多
+[Pod里的容器间通讯](/docs/tasks/configure-pod-container/communicate-containers-same-pod/).
```
Comment (jimexist) [inline docs/tasks/configure-pod-container/configure-pod-initialization.md:180]: 原文有问题 不用换行

**Review by jimexist — APPROVED** [review]

**Pull request merged**